lib/versions/version-info.js

```diff
@@ -151,7 +151,9 @@ var getCdnVersion = function() {
           var cdnResult = shell.exec('curl http://ajax.googleapis.com/ajax/libs/angularjs/' + version + '/angular.min.js ' +
                     '--head --write-out "%{http_code}" -o /dev/null -silent',
                                       {silent: true});
-          if (cdnResult.code === 0) {
+          // Ignore if Curl can't write the file (code 23), e.g. on Windows, as we still get
+          // the needed output from --write-out
+          if (cdnResult.code === 0 || cdnResult.code === 23) {
             var statusCode = cdnResult.output.trim();
```

Comment:
Really we only care about whether requests to this URL are 404 or not, right?

Why don't we ditch the curl altogether and just use the `http` library?

https://nodejs.org/dist/latest-v4.x/docs/api/http.html#http_http_request_options_callback